.github/workflows/build.yml

```diff
@@ -44,28 +44,39 @@ jobs:
             .*/**/*
             !readme.txt
             !changelog.txt
+
+      - name: Prevent git tag versioning
+        shell: bash
+        run: npm config set git-tag-version false
+
       - name: Install Dependencies
         if: github.event.pull_request.title == null || steps.changelog_diff_files.outputs.diff
-        run: npm ci
-      - name: Build and Deploy
-        if: github.event.pull_request.title == null || steps.changelog_diff_files.outputs.diff
+        uses: ./.github/workflows/install-dependencies
+
+      - name: Set ENV variables
+        shell: bash
         run: |
-          npm config set git-tag-version false
-          export PLUGIN_VERSION=$(date '+%Y%m%d.%H%M')
-          export PACKAGE_VERSION=$(cat package.json \
-            | grep version \
-            | head -1 \
-            | awk -F: '{ print $2 }' \
-            | sed 's/[",]//g' \
-            | tr -d '[[:space:]]')
-          export PLUGIN_FOLDER_FILENAME="elementor-${PACKAGE_VERSION}-${PLUGIN_VERSION}"
-          export PLUGIN_ZIP_FILENAME="${PLUGIN_FOLDER_FILENAME}.zip"
-          npx grunt build
-          sed -i -E "s/Version: (.*?)/Version: ${PACKAGE_VERSION}-${PLUGIN_VERSION}/g" build/elementor.php
-          sed -i -E "s/ELEMENTOR_VERSION', '(.*?)'/ELEMENTOR_VERSION', '${PACKAGE_VERSION}-${PLUGIN_VERSION}'/g" build/elementor.php
-          mv build elementor
-          zip -r $PLUGIN_ZIP_FILENAME elementor
-          echo "PLUGIN_FOLDER_FILENAME=${PLUGIN_FOLDER_FILENAME}" >> $GITHUB_ENV
+          DATE_VERSION=$(date '+%Y%m%d.%H%M')
+          VERSION=$(cat package.json \
+              | grep version \
+              | head -1 \
+              | awk -F: '{ print $2 }' \
+              | sed 's/[",]//g' \
+              | tr -d '[[:space:]]')
+
+          PACKAGE_VERSION=${VERSION}-${DATE_VERSION}
+          PLUGIN_FOLDER_FILENAME="elementor-${PACKAGE_VERSION}"
+
+          echo "PACKAGE_VERSION=$PACKAGE_VERSION" >> $GITHUB_ENV
+          echo "PLUGIN_FOLDER_FILENAME=$PLUGIN_FOLDER_FILENAME" >> $GITHUB_ENV
+
+      - name: Build plugin
+        uses: ./.github/workflows/build-plugin
+        with:
+          PACKAGE_VERSION: ${{ env.PACKAGE_VERSION }}
+          BUILD_SCRIPT_PATH: "./.github/scripts/build-zip.sh"
+          UPDATE_VERSIONS_SCRIPT_PATH: "./.github/scripts/update-version-elementor-php.sh"
```

Comment:
Why is this required?

Reply:
The code that updates the plugin version exists in `bump-channel-version` workflow.
https://github.com/elementor/elementor/blob/main/.github/workflows/bump-channel-version/action.yml#L41-L42
Please pay attention that don't increment the version twice in the same build process.

Reply:
AFAIU, this PR aligns the build for all CI & release processes, no?
The bumps/versions/whatever are irrelevant for CI, aren't they?